pyproject.toml

```diff
@@ -13,13 +13,13 @@ python = "^3.11"
 pyyaml = "^6.0.2"
 xarray = "^2024.7.0"
 netcdf4 = "^1.7.1"
-cfgrib = "^0.9.14.0"
 couchbase = "^4.3.1"
 pyproj = "^3.6.1"
 numpy = "^2.1.0"
 metpy = "^1.6.3"
 prometheus-client = "^0.20.0"
 
+cfgrib = "^0.9.14.1"
```

Comment:
An optional nit - it'd be nice to just update cfgrib's version in place here instead of moving it to the bottom of the `[tool.poetry.dependencies]` section. E.g. instead of the above diff, just doing:

```diff
[tool.poetry.dependencies]
python = "^3.11"
pyyaml = "^6.0.2"
xarray = "^2024.7.0"
netcdf4 = "^1.7.1"
+ cfgrib = "^0.9.14.1"
couchbase = "^4.3.1"
pyproj = "^3.6.1"
numpy = "^2.1.0"
metpy = "^1.6.3"
prometheus-client = "^0.20.0"
```

Reply:
ok